### PR TITLE
Colo variables update in styles scss file

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
# Issue Tracking

JIRA:  https://aottech.atlassian.net/browse/FWF-2573
Issue Type: BUG/ FEATURE

# Changes
variables updated with styles.scss file

### CODE REVIEW

File: forms-flow-web/src/styles.scss [hunk 1]
```diff
@@ -49,22 +13,22 @@ $dark: $gray-800;
 }
 
 .text-primary {
-  color: $primary;
+  color: var(--color-info);
 }
```
Comment (fahad-aot) [inline forms-flow-web/src/styles.scss:17]: I have doubt that text-primary class already defined on MF with var(--color-primary) i think.so this one does make any sense

Comment (Josephalexantony-aot) [reply forms-flow-web/src/styles.scss:17]: good catch.

Comment (shuhaib-aot) [reply forms-flow-web/src/styles.scss:17]: yes. text-primary class need to bring inside theme service/bootstrapOverride and need to use color-primary

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/styles.scss (hunk 1 above)
Comment (Josephalexantony-aot) [reply forms-flow-web/src/styles.scss:17]: good catch @fahad-aot 
The .text-primary class is already declared in Global.scss file and that will override this class and as you said it doesn't make any impact. 
Need to remove all the unwanted redundant classes from styles.scss, may be with another ticket.

Comment (fahad-aot) [reply forms-flow-web/src/styles.scss:17]: okay.

**Review by fahad-aot — APPROVED** [review]

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**